src/pages/policy/output/Analysis.jsx

```diff
@@ -59,14 +70,44 @@ export default function Prompt(props) {
   });
   const openai = new OpenAIApi(configuration);
   const [showPrompt, setShowPrompt] = useState(false);
-  const lines = analysisPrompt.split("\n");
+  const lines = prompt.split("\n");
+
+  const handleAudienceChange = (audienceValue) => {
+    setAudience(audienceValue);
+  };
+
+
+  const buttonWidth = "80px";
+  const activeColor = "#6c757d";
+  const inactiveColor = "white";
+  const borderColor = "1px solid #6c757d";
+
+  const AudienceButton = ({ audienceValue, currentAudience, handleAudienceChange }) => (
```

Comment:
We do use functional components in the codebase, but all the others use the same `function AudienceButton() {}` syntax.